pkg/registry/experimental/controller/etcd/etcd.go

```diff
@@ -105,10 +92,17 @@ func (r *ScaleREST) Update(ctx api.Context, obj runtime.Object) (runtime.Object,
 	if err != nil {
```

Comment:
previously, it was possible to send a scale request without a resource version, and the server would get/update. that seems like it would still be useful ("I want to scale to 10... I don't care what it currently is"). How would that be expressed?